README.md

```diff
@@ -475,6 +475,8 @@ Note: For admission webhooks registered for DELETE operations, use Kubernetes v1
     - DELETE
 ```
 
+If you use the helm chart, set `enableDeleteOperations=true`.
```

Comment:
We should update readme after release is cut

Reply:
will that happen automatically based on the readme in `staging_docs`?